packages/flutter/lib/src/widgets/drag_target.dart

```diff
@@ -229,6 +236,17 @@ class Draggable<T> extends StatefulWidget {
   /// callback is still in the tree.
   final VoidCallback onDragCompleted;
 
+  /// Called when the draggable is dropped, returning its current
+  /// [DraggableDetails].
+  ///
+  /// This function might be called after this widget has been removed from the
```

Comment:
What's the benefit of having this be called even after the widget has been removed from the tree? Because if there's not a clear benefit, it seems to me that you'd be better of not creating the situation in the first place: it seems like a convenient way to shoot yourself in the foot that could be avoided by just not calling the callback if this widget has been removed.

Reply:
Ok, I can adjust it so that it doesn't invoke it if the widget was removed from the tree.